source/_components/sensor.vasttrafik.markdown

```diff
@@ -23,22 +23,48 @@ Add the data to your `configuration.yaml` file as shown in the example:
 # Example configuration.yaml entry
 sensor:
   - platform: vasttrafik
-    key: XXXXXXXXXXXXXXXXXXX
-    secret: YYYYYYYYYYYYYYYYY
+    key: YOUR_API_KEY
+    secret: YOUR_API_SECRET
     departures:
       - from: Musikvägen
 ```
 
-Configuration variables:
-
-- **key** (*Required*): The API key to access your Västtrafik account.
-- **secret** (*Required*): The API secret to access your Västtrafik account.
-- **departures** array (*Required*): List of travel routes.
-  - **name** (*Optional*): Name of the route.
-  - **from** (*Required*): The start station.
-  - **heading** (*Optional*): Direction of the traveling.
-  - **lines** (*Optional*): Only consider these lines.
-  - **delay** (*Optional*): Delay in minutes. Defaults to 0.
+{% configuration %}
+key:
+  description: The API key to access your Västtrafik account.
+  required: true
+  type: string
+secret:
+  description: The API secret to access your Västtrafik account.
+  required: true
+  type: string
+departures:
+  description: List of travel routes.
+  required: true
+  type: map
+  keys:
+    name:
+      description: Name of the route.
+      required: false
+      type: string
+    from:
+      description: The start station.
+      required: true
+      type: string
+    heading:
+      description: Direction of the traveling.
+      required: false
+      type: string
+    lines:
+      description: Only consider these lines.
+      required: false
+      type: string
```

Comment:
I think it is supposed to be described as a list of strings (`list, string`), but home-assistant/developers.home-assistant#104 is not resolved yet.